HelpSource/Reference/AudioDeviceSelection.schelp

```diff
@@ -138,31 +138,31 @@ LIST::
 ## WASAPI
 ||
 LIST::
-## strong::Maximum channel count: :: Usually supports mono or stereo only
+## strong::Maximum channel count: :: Usually supports stereo only, depending on the driver
```

Comment:
```suggestion
## strong::Maximum channel count: :: Usually supports mono or stereo only. Devices with more than 2 channels might be represented as multiple stereo pairs.
```

Reply:
I like your suggestion and will apply it.

Re: mono - I have not myself seen a device that's reported as mono. However, that wording was suggested by someone else IIRC so maybe it should stay...?

Reply:
yeah, in the absence of info to the contrary let's leave it in